src/BenchmarksClient/Workers/H2LoadWorker.cs

```diff
@@ -51,14 +58,26 @@ private void InitializeJob()
         public async Task StartJobAsync(ClientJob job)
         {
             _job = job;
+
+            if (job.ClientProperties.TryGetValue("RequestBody", out var requestBodyText))
```

Comment:
I would just put this block in `StartProcess`

You get bonus points if you handle it on wrk, wrk2 and bombardier ;)